src/uikit.cs

```diff
@@ -16085,4 +16153,256 @@ interface UITextAttributesConstants {
+		bool AllowsMoveOperation { get; }
+	
+		[Abstract]
+		[Export ("restrictedToDraggingApplication")]
+		bool RestrictedToDraggingApplication { [Bind ("isRestrictedToDraggingApplication")] get; }
+	
+		[Abstract]
+		[Export ("hasItemsConformingToTypeIdentifiers:")]
+		bool HasItemsConformingToTypeIdentifiers (string[] typeIdentifiers);
+	
+		[Abstract]
+		[Export ("canLoadObjectsOfClass:")]
+		bool CanLoadObjectsOfClass (INSItemProviderReading aClass);
+	}
+	
+	[NoWatch, NoTV, iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface UIDragItem
+	{
+		[Export ("initWithItemProvider:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSItemProvider itemProvider);
+	
+		[Export ("itemProvider")]
+		NSItemProvider ItemProvider { get; }
+	
+		[NullAllowed, Export ("localObject", ArgumentSemantic.Strong)]
+		NSObject LocalObject { get; set; }
+	
+		[NullAllowed, Export ("previewProvider", ArgumentSemantic.Copy)]
+		Func<UIDragPreview> PreviewProvider { get; set; }
+	}
+	
+	[NoWatch, NoTV, iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface UIDragPreview : NSCopying
+	{
+		[Export ("initWithView:parameters:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (UIView view, UIDragPreviewParameters parameters);
+	
+		[Export ("initWithView:")]
+		IntPtr Constructor (UIView view);
+	
+		[Export ("view")]
+		UIView View { get; }
+	
+		[Export ("parameters", ArgumentSemantic.Copy)]
+		UIDragPreviewParameters Parameters { get; }
+	}
+	
+	[NoWatch, NoTV, iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	interface UIDragPreviewParameters : NSCopying
+	{
+		[Export ("initWithTextLineRects:")]
+		IntPtr Constructor (NSValue[] textLineRects);
+	
+		[NullAllowed, Export ("visiblePath", ArgumentSemantic.Copy)]
+		UIBezierPath VisiblePath { get; set; }
+	
+		[Export ("backgroundColor", ArgumentSemantic.Copy)]
+		UIColor BackgroundColor { get; set; }
+	}
+	
+	[NoWatch, NoTV, iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface UIDragPreviewTarget : NSCopying
+	{
+		[Export ("initWithContainer:center:transform:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (UIView container, CGPoint center, CGAffineTransform transform);
+	
+		[Export ("initWithContainer:center:")]
+		IntPtr Constructor (UIView container, CGPoint center);
+	
+		[Export ("container")]
+		UIView Container { get; }
+	
+		[Export ("center")]
+		CGPoint Center { get; }
+	
+		[Export ("transform")]
+		CGAffineTransform Transform { get; }
+	}
+	
+	[NoWatch, NoTV, iOS (11,0)]
+	[Protocol]
+	interface UIDragSession : UIDragDropSession
+	{
+		[Abstract]
+		[NullAllowed, Export ("localContext", ArgumentSemantic.Strong)]
+		NSObject LocalContext { get; set; }
+	}
+	
+	[NoWatch, NoTV, iOS (11,0)]
+	[BaseType (typeof(NSObject))] // If Apple adds a delegate setter: Delegates=new string [] {"Delegate"}, Events=new Type [] { typeof (UIDropInteractionDelegate)})]
+	[DisableDefaultCtor]
+	interface UIDropInteraction : IUIInteraction
```

Comment:
Here you want to use `UIInteraction` interaction instead or the generator will pick the wrong interface

`interface UIDropInteraction : UIInteraction `